pygount/analysis.py

```diff
@@ -706,8 +728,13 @@ def encoding_for(source_path: str, encoding: str = "automatic", fallback_encodin
     assert encoding is not None
 
     if encoding == "automatic":
-        with open(source_path, "rb") as source_file:
-            heading = source_file.read(128)
+        if file_handle is None:
+            with open(source_path, "rb") as source_file:
+                heading = source_file.read(128)
+        else:
+            file_handle.seekable and file_handle.seek(0)
+            heading = file_handle.read(128)
+            file_handle.seekable and file_handle.seek(0)
```

Comment:
What happens if the `file_handle` is not seekable? Wouldn't this exclude the initial 128 bytes from the analysis?

Reply:
Will be addressed with #106